From b545a036f1de8799ce61495ef50a3a6640d37433 Mon Sep 17 00:00:00 2001
From: Gabriel Craciunescu <crazy@frugalware.org>
Date: Wed, 26 Apr 2017 09:42:29 +0200
Subject: [PATCH] kcm_networkmanagement.desktop sort out

	Change X-KDE-System-Settings-Parent-Category= to connectivity
	since the category itself is called Network and this seems really
	ugly to have 2 'connections' shown in systemsettings.
	Also changed Name= to be 'Network Connections'.

	I think it is meant to look like this :
			Category Network:
				Sub-Category: Connectivity
					KCM-Name: Samba Shares
					KCM-Name: Network Connections
					<and so on>

Signed-off-by: Gabriel Craciunescu <crazy@frugalware.org>
---
 kcm/kcm_networkmanagement.desktop | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/kcm/kcm_networkmanagement.desktop b/kcm/kcm_networkmanagement.desktop
index bbc980fb..60e73b8e 100755
--- a/kcm/kcm_networkmanagement.desktop
+++ b/kcm/kcm_networkmanagement.desktop
@@ -5,15 +5,15 @@ Type=Service
 X-KDE-ServiceTypes=KCModule
 X-KDE-Library=kcm_networkmanagement
 X-KDE-ParentApp=kcontrol
-X-KDE-System-Settings-Parent-Category=network
+X-KDE-System-Settings-Parent-Category=connectivity
 X-KDE-Weight=40
 
-Name=Connections
+Name=Network Connections
 Name[ca]=Connexions
 Name[ca@valencia]=Connexions
 Name[cs]=Spojení
 Name[da]=Forbindelser
-Name[de]=Verbindungen
+Name[de]=Netzwerkverbindungen
 Name[el]=Συνδέσεις
 Name[en_GB]=Connections
 Name[es]=Conexiones
-- 
2.12.2

